site-config.yml

```
@@ -3,4 +3,5 @@ hub: "cdcepi/FluSight-forecast-hub/"
 title: "Example Hub Dashboard Website"
 pages:
   - "about.md"
+  - "data.md"
```

You named the file `data.qmd`, so it needs to match here. 

```suggestion
  - "data.qmd"
```